exo-add/helpers/helpers.go

```diff
@@ -0,0 +1,107 @@
+package helpers
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/Originate/exosphere/exo-add/os_helpers"
+	"github.com/Originate/exosphere/exo-add/types"
+	"github.com/Originate/exosphere/exo-add/user_input"
+	"github.com/tmrts/boilr/pkg/template"
+	"gopkg.in/yaml.v2"
+)
+
+func CheckForService(serviceRole string, existingServices []string) {
```

Comment:
The name `CheckXXX` implies that this method just checks for something. Since this method also exits the app, let's call it `VerifyServiceDoesNotExist`.